asciidoctorj-arquillian-extension/build.gradle

```diff
@@ -0,0 +1,28 @@
+dependencies {
+  compile project(path: ':asciidoctorj')
+  compile project(path: ':asciidoctorj-test-support', configuration: 'tests')
+  compile "org.jboss.arquillian.container:arquillian-container-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-impl-base:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-impl-base:$arquillianVersion"
+}
+
+jar.dependsOn project(':asciidoctorj').testClasses
+
+jar {
+  from project(':asciidoctorj').sourceSets.test.output
+  include 'org/asciidoctor/arquillian/**/*'
```

Comment:
BTW: I am a little disappointed that you always need some extra plugins to fully cover Maven's possibilities. 
While I prefer to not export test deps I just can't understand why there has to be a pile of gradle plugins for provided and optional. Nebula is one and there is at least another one from spring that does probably the same.
But that's just my own opinion. 



> Am 06.06.2015 um 14:11 schrieb Frank Becker <notifications@github.com>:
> 
> In asciidoctorj-arquillian-extension/build.gradle:
> 
> > @@ -0,0 +1,28 @@
> > +dependencies {
> > +  compile project(path: ':asciidoctorj')
> > +  compile project(path: ':asciidoctorj-test-support', configuration: 'tests')
> > +  compile "org.jboss.arquillian.container:arquillian-container-spi:$arquillianVersion"
> > +  compile "org.jboss.arquillian.container:arquillian-container-test-spi:$arquillianVersion"
> > +  compile "org.jboss.arquillian.container:arquillian-container-impl-base:$arquillianVersion"
> > +  compile "org.jboss.arquillian.container:arquillian-container-test-impl-base:$arquillianVersion"
> > +}
> > +
> > +jar.dependsOn project(':asciidoctorj').testClasses
> > +
> > +jar {
> > +  from project(':asciidoctorj').sourceSets.test.output
> > +  include 'org/asciidoctor/arquillian/**/*'
> Hmm....that's an idea...
> 
> Well, there is another nebula plugin for that. 
> https://github.com/nebula-plugins/gradle-extra-configurations-plugin
> 
> —
> Reply to this email directly or view it on GitHub.
>